README.md

```diff
@@ -218,6 +220,17 @@ running job or a known temperamental pod on a node will stop it rebooting.
 > up a RebootRequired alert as described in the next section so that
 > you can intervene manually if reboots are blocked for too long.
 
+### Adding node labels before and after reboots
+
+If you need to add node labels before and after the reboot process, you can use `--pre-reboot-node-labels` and `--after-reboot-node-labels`:
+
+```console
+      --pre-reboot-node-labels=zalando=notready
+      --after-reboot-node-labels=zalando=ready
+```
+
+Note that label keys specified by these two flags should match. If they do not match, a warning will be generated.
```

Comment:
Please add a note, that the labels could be comma-delimited or the flags could be added multiple times.

Reply:
Done.